payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/NameConfig.kt

```diff
@@ -24,7 +24,7 @@ class NameConfig : TextFieldConfig {
 
     override fun determineState(input: String): TextFieldState {
         return when {
-            input.isEmpty() -> Error.Blank
+            input.isBlank() -> Error.Blank
```

Comment:
@michelleb-stripe using `isBlank` here instead of `isEmpty`, since `isBlank` is a super set and covers the case where a user enters a bunch of spaces.